icons/pulsar/pulsar-original-wordmark.svg

```diff
@@ -0,0 +1 @@
 9.316-4.129 4.121-8.996 6.965-13.727 8.039-3.543.805-6.641.516-9.016-.785l-1.414 1.188c1.859 1.188 4.125 1.793 6.66 1.793 1.324 0 2.715-.156 4.152-.492 5.043-1.145 10.227-4.143 14.578-8.486 2.555-2.555 4.688-5.441 6.234-8.422 1.75 5.195 1.322 10.055-1.639 13.012-2.242 2.242-5.766 3.07-9.824 2.418a25.38 25.38 0 0 1-2.309 1.277c1.727.449 3.387.693 4.934.693 3.41 0 6.35-1.072 8.436-3.154 5.426-5.422 3.609-16.195-3.859-24.949zm-3.428 2.795l-.586.227a2.91 2.91 0 0 1-.785 1.566c-.09.09-.18.156-.289.246a8.32 8.32 0 0 1 .852 3.652c-.023 4.633-3.793 8.375-8.41 8.375a8.24 8.24 0 0 1-3.656-.852c-.07.09-.137.203-.227.289a2.77 2.77 0 0 1-1.57.785l-.223.582c1.66 1.031 3.609 1.637 5.695 1.637 5.969 0 10.836-4.859 10.836-10.816a10.74 10.74 0 0 0-1.637-5.691zm46.43.023v12.742h10.363v-2.195h-8.168V63.789H77.16zm13.793 0a2.38 2.38 0 0 0-1.703.695 2.28 2.28 0 0 0-.695 1.699v2.668a2.28 2.28 0 0 0 .695 1.699c.469.473 1.031.693 1.703.693h6.93c.09 0 .203.068.203.229v2.664c0 .156-.113.223-.203.223h-6.93a.21.21 0 0 1-.223-.223v-.898h-2.176v.898c0 .672.221 1.23.693 1.703a2.3 2.3 0 0 0 1.703.689h6.932a2.35 2.35 0 0 0 1.684-.689c.473-.473.719-1.031.719-1.703v-2.664a2.31 2.31 0 0 0-.719-1.703 2.28 2.28 0 0 0-1.684-.695h-6.932a.21.21 0 0 1-.223-.223v-2.668a.21.21 0 0 1 .223-.223h6.932c.09 0 .203.066.203.223v.896h2.197v-.896a2.3 2.3 0 0 0-.719-1.699c-.449-.449-1.029-.695-1.682-.695h-6.93zm13.906 0a2.38 2.38 0 0 0-1.703.695 2.28 2.28 0 0 0-.693 1.699v10.328h2.176v-4.35h7.354v4.35h2.199V66.184a2.3 2.3 0 0 0-.719-1.699 2.43 2.43 0 0 0-1.703-.695h-6.91zm-54.996.023v12.699h2.176v-4.414h.156l6.977-.023c.672 0 1.23-.246 1.703-.715s.719-1.031.719-1.68v-3.473c0-.648-.246-1.23-.719-1.68-.449-.469-1.031-.715-1.684-.715h-9.328zm13.391 0v10.324c0 .672.221 1.23.693 1.703a2.3 2.3 0 0 0 1.703.691h6.93a2.35 2.35 0 0 0 1.686-.691c.473-.473.719-1.031.719-1.703V63.813h-2.199v10.324c0 .156-.113.223-.203.223h-6.93a.21.21 0 0 1-.223-.223V63.813h-2.176zm52.957.018v12.699h2.176v-4.434h3.363l.023.047 3.699 4.387h2.445v-.469l-3.32-3.941h.896a2.46 2.46 0 0 0 1.729-.736 2.3 2.3 0 0 0 .719-1.68V66.23c0-.648-.246-1.23-.719-1.68a2.37 2.37 0 0 0-1.703-.721h-9.309zm-94.699.428a5.21 5.21 0 0 0-5.207 5.199 5.21 5.21 0 0 0 5.207 5.195c2.871 0 5.203-2.332 5.203-5.195a5.21 5.21 0 0 0-5.203-5.199zm83.328 1.727a.21.21 0 0 1 .021 0h6.91a.21.21 0 0 1 .223.223l-.002 3.809h-7.354v-3.809a.21.21 0 0 1 .201-.223zm13.75.021a.21.21 0 0 1 .02.002h6.91a.21.21 0 0 1 .223.223v3.473a.21.21 0 0 1-.223.223h-6.91a.21.21 0 0 1-.223-.223V66.23a.21.21 0 0 1 .203-.225zm-66.305.002h6.906c.137 0 .227.09.227.223v3.473c0 .133-.09.225-.227.225h-6.906c-.137 0-.227-.092-.227-.225V66.23c0-.133.09-.223.227-.223zm-36.652 7.949c-.359 0-.719.133-.984.402-.406.402-.52 1.008-.293 1.5l-2.781 1.074L0 90.688l.246.246L14.02 79.375l1.098-2.777a1.49 1.49 0 0 0 .539.113 1.34 1.34 0 0 0 .965-.402c.539-.539.539-1.41 0-1.949a1.39 1.39 0 0 0-.988-.402zm-1.254 6.941l-1.371 1.164a33.49 33.49 0 0 0 3.746 2.777l1.953-.852c-1.594-.918-2.871-1.855-4.328-3.09z"/></svg>
```

Comment:
`height` and `width` can be removed. the `fill` color can be changed to `#2c3e50`
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#2c3e50" viewBox="0 0 128 128"><path d="M42.797 48L28.98 59.555l-1.078 2.777c-.18-.066-.359-.109-.516-.109a1.34 1.34 0 0 0-.965.402c-.539.535-.539 1.41 0 1.949a1.41 1.41 0 0 0 .986.402c.359 0 .719-.137.984-.402.406-.406.52-1.008.293-1.5l2.781-1.078 11.576-13.75-.246-.246zm-12.795 1.846c-1.431-.036-2.954.122-4.543.482-1.324.293-2.645.719-3.969 1.277-6.416-2.622-12.697-2.329-16.443 1.41-2.895 2.891-3.859 7.348-2.691 12.52.293 1.32.738 2.645 1.277 3.941-2.242 5.512-2.355 10.93.023 14.715l1.188-1.41c-1.613-2.934-1.613-6.898-.199-11.086a32.11 32.11 0 0 0 4.262 6.271l1.143-1.346c-1.906-2.262-3.43-4.68-4.527-7.145 1.434-3.223 3.656-6.473 6.594-9.406 2.871-2.867 6.102-5.105 9.398-6.582 2.422 1.074 4.867 2.574 7.176 4.523l1.348-1.145a33.1 33.1 0 0 0-6.305-4.23c.719-.246 1.414-.449 2.109-.605 3.543-.805 6.641-.516 9.016.785l1.414-1.186c-1.753-1.121-3.884-1.725-6.27-1.785zM13.66 51.584c1.774.025 3.678.375 5.629 1.029-2.961 1.523-5.855 3.629-8.434 6.227-2.625 2.621-4.711 5.508-6.211 8.422-.227-.695-.449-1.391-.605-2.105-1.012-4.594-.227-8.463 2.219-10.904v-.002c1.851-1.848 4.446-2.708 7.402-2.666zm25.686 3.135l-1.17 1.434c.988 1.813 1.367 4.074 1.145 6.605a28.54 28.54 0 0 1 1.393 2.439c.852-4.098.379-7.726-1.367-10.479zm-17.834 3.918a10.85 10.85 0 0 0-10.836 10.818c0 2.082.605 4.032 1.637 5.689l.586-.227a2.91 2.91 0 0 1 .785-1.566c.09-.09.18-.156.289-.246a8.31 8.31 0 0 1-.852-3.648c0-4.617 3.77-8.379 8.391-8.379a8.32 8.32 0 0 1 3.656.852c.066-.09.133-.199.223-.289a2.77 2.77 0 0 1 1.57-.785l.246-.582c-1.66-1.031-3.609-1.637-5.695-1.637zm12.646 2.334l-1.162 1.342c1.926 2.305 3.453 4.746 4.527 7.145l-.02.066c-1.48 3.293-3.748 6.516-6.553 9.316-4.129 4.121-8.996 6.965-13.727 8.039-3.543.805-6.641.516-9.016-.785l-1.414 1.188c1.859 1.188 4.125 1.793 6.66 1.793 1.324 0 2.715-.156 4.152-.492 5.043-1.145 10.227-4.143 14.578-8.486 2.555-2.555 4.688-5.441 6.234-8.422 1.75 5.195 1.322 10.055-1.639 13.012-2.242 2.242-5.766 3.07-9.824 2.418a25.38 25.38 0 0 1-2.309 1.277c1.727.449 3.387.693 4.934.693 3.41 0 6.35-1.072 8.436-3.154 5.426-5.422 3.609-16.195-3.859-24.949zm-3.428 2.795l-.586.227a2.91 2.91 0 0 1-.785 1.566c-.09.09-.18.156-.289.246a8.32 8.32 0 0 1 .852 3.652c-.023 4.633-3.793 8.375-8.41 8.375a8.24 8.24 0 0 1-3.656-.852c-.07.09-.137.203-.227.289a2.77 2.77 0 0 1-1.57.785l-.223.582c1.66 1.031 3.609 1.637 5.695 1.637 5.969 0 10.836-4.859 10.836-10.816a10.74 10.74 0 0 0-1.637-5.691zm46.43.023v12.742h10.363v-2.195h-8.168V63.789H77.16zm13.793 0a2.38 2.38 0 0 0-1.703.695 2.28 2.28 0 0 0-.695 1.699v2.668a2.28 2.28 0 0 0 .695 1.699c.469.473 1.031.693 1.703.693h6.93c.09 0 .203.068.203.229v2.664c0 .156-.113.223-.203.223h-6.93a.21.21 0 0 1-.223-.223v-.898h-2.176v.898c0 .672.221 1.23.693 1.703a2.3 2.3 0 0 0 1.703.689h6.932a2.35 2.35 0 0 0 1.684-.689c.473-.473.719-1.031.719-1.703v-2.664a2.31 2.31 0 0 0-.719-1.703 2.28 2.28 0 0 0-1.684-.695h-6.932a.21.21 0 0 1-.223-.223v-2.668a.21.21 0 0 1 .223-.223h6.932c.09 0 .203.066.203.223v.896h2.197v-.896a2.3 2.3 0 0 0-.719-1.699c-.449-.449-1.029-.695-1.682-.695h-6.93zm13.906 0a2.38 2.38 0 0 0-1.703.695 2.28 2.28 0 0 0-.693 1.699v10.328h2.176v-4.35h7.354v4.35h2.199V66.184a2.3 2.3 0 0 0-.719-1.699 2.43 2.43 0 0 0-1.703-.695h-6.91zm-54.996.023v12.699h2.176v-4.414h.156l6.977-.023c.672 0 1.23-.246 1.703-.715s.719-1.031.719-1.68v-3.473c0-.648-.246-1.23-.719-1.68-.449-.469-1.031-.715-1.684-.715h-9.328zm13.391 0v10.324c0 .672.221 1.23.693 1.703a2.3 2.3 0 0 0 1.703.691h6.93a2.35 2.35 0 0 0 1.686-.691c.473-.473.719-1.031.719-1.703V63.813h-2.199v10.324c0 .156-.113.223-.203.223h-6.93a.21.21 0 0 1-.223-.223V63.813h-2.176zm52.957.018v12.699h2.176v-4.434h3.363l.023.047 3.699 4.387h2.445v-.469l-3.32-3.941h.896a2.46 2.46 0 0 0 1.729-.736 2.3 2.3 0 0 0 .719-1.68V66.23c0-.648-.246-1.23-.719-1.68a2.37 2.37 0 0 0-1.703-.721h-9.309zm-94.699.428a5.21 5.21 0 0 0-5.207 5.199 5.21 5.21 0 0 0 5.207 5.195c2.871 0 5.203-2.332 5.203-5.195a5.21 5.21 0 0 0-5.203-5.199zm83.328 1.727a.21.21 0 0 1 .021 0h6.91a.21.21 0 0 1 .223.223l-.002 3.809h-7.354v-3.809a.21.21 0 0 1 .201-.223zm13.75.021a.21.21 0 0 1 .02.002h6.91a.21.21 0 0 1 .223.223v3.473a.21.21 0 0 1-.223.223h-6.91a.21.21 0 0 1-.223-.223V66.23a.21.21 0 0 1 .203-.225zm-66.305.002h6.906c.137 0 .227.09.227.223v3.473c0 .133-.09.225-.227.225h-6.906c-.137 0-.227-.092-.227-.225V66.23c0-.133.09-.223.227-.223zm-36.652 7.949c-.359 0-.719.133-.984.402-.406.402-.52 1.008-.293 1.5l-2.781 1.074L0 90.688l.246.246L14.02 79.375l1.098-2.777a1.49 1.49 0 0 0 .539.113 1.34 1.34 0 0 0 .965-.402c.539-.539.539-1.41 0-1.949a1.39 1.39 0 0 0-.988-.402zm-1.254 6.941l-1.371 1.164a33.49 33.49 0 0 0 3.746 2.777l1.953-.852c-1.594-.918-2.871-1.855-4.328-3.09z"/></svg>
```